app/views/admin/organizations/_form.html.haml

```diff
@@ -73,7 +91,19 @@
     .form-group
       = f.check_box :approved
       = f.label :approved, 'Validated & approved'
+
+    .form-group
```

Comment:
I know this is additional complexity, but I think I would prefer if `regional` was actually a `paid_feature_slug`, and the regional organizations were just the organizations that fall within the organization search radius.

So you would add the search_radius, but otherwise, you wouldn't need to select the organizations and update them. This seems like a less labor intensive process, and less potentially error prone. It also means we'll focus on adding locations to organizations, rather than adding regional organizations (which is what I'd prefer was focused on).

I would either add another column to the db like `child_ids`, or rename `child_ids`, and switch it to a structure like `suborganizations: { child_ids: [], regional_ids: [] }`. Or something similar!

Reply:
Potentially related to https://github.com/bikeindex/bike_index/pull/1404#discussion_r345238664 - if we're associating based on geocoding and locations, it might make sense to skip adding the address inputs to the organization - and instead just geocode based on the first location for the organization (with some note on the organization form around "search radius" that makes it explicit what is happening).

Just an idea, not a requirement.

Reply:
Makes sense — updated in 2656c690
What's the intent with adding a `regional_ids` column? (as opposed to querying for those orgs, that is)

Reply:
Just a guess that the geospatial query to select the regional organizations was an expensive query. Probably doesn't matter, caching can be resolved later.